doc/sphinx/practical-tools/utilities.rst

```diff
@@ -83,31 +85,37 @@ project because it makes it easy to track the history of your changes.
 `git <https://git-scm.com/>`_ is the system most used by Coq projects.
 Typically, each package has its own git repository.
 
-**Single directory projects.**  If your project has a single file (and
-depends only on packages in the Coq installation), put it in any directory
-you like and run Coq from that directory.  If your project has multiple files,
-then you need to compile them in an order that matches their dependencies.
-If B.v depends on A.v, then you should compile A.v before B.v.  You can do this
-with :n:`coqc A.v` followed by :n:`coqc B.v`, but you may find it tedious to manage
-the dependencies.  You may find it easier to treat your project as a
-multi-directory project.
-
-**Multi-directory projects.**  For these, we recommend using `_CoqProject` files
-to define one or more packages for your project.  With that, you can automatically
-generate a makefile that matches the dependencies.
 A :gdef:`package` is a group of files in a top directory and its subdirectories
 that's built and installed as a unit, just like the opam packages mentioned above.
-`_CoqProject` must be in the top directory.
 
-Large projects that have multiple packages (either their own code or dependencies
-on packages that you've not installed as user-contributed packages) need
-additional setup.  The :ref:`COQPATH <COQPATH>` environment variable is good mechanism
-for such projects.
+For a project that has only a single file, you can create the file wherever you like
+and then step through it one of the Coq IDEs.
 
+If your project has multiple files in a single directory that depend on each
+other through :cmd:`Require` commands, they must be compiled in an order that
+matches their dependencies.
 Scripts in `.v` files must be compiled to `.vo` files using `coqc` before they
-can be :cmd:`Require`\d in other files.  Currently, each `.vo` file is created in
-the same directory as its `.v` file.  Furthermore, Coq has to know the :term:`logical
-path` to associate with the directories containing your compiled files.
+can be :cmd:`Require`\d in other files.  Currently, the `.vo` file is created in
+the same directory as its `.v` file.  For example,
+if B.v depends on A.v, then you should compile A.v before B.v.  You can do this
+with :n:`coqc A.v` followed by :n:`coqc B.v`, but you may find it tedious to
+manage the dependencies, particularly as the number of files increases.
+
+If your project files are in multiple directories, you would also need to pass
+additional command-line -Q and -R parameters to your IDE.  Too much detail.
+
+Instead, by creating a `_CoqProject` file, you can automatically generate
```

Comment:
I'd remove instead here and just start with "By creating a".

Reply:
I'm contrasting using _CoqProject with what comes before.